docs/cep/windows/window-types/tumbling.md

```diff
@@ -0,0 +1,29 @@
+---
+title: WINDOW TUMBLING()
+---
+
+A window that holds an incoming events batch. When a new set of events arrives, the old events are expired. Tumbling windows can be used to aggregate events that comes in batches. If it has the parameter length specified, then the batch window processes the batch as several chunks.
+
+## Syntax
+
+    WINDOW TUMBLING()
+    WINDOW TUMBLING(length <INT>)
+
+## Query Parameters
+
+| Name          | Description           | Default Value               | Possible Data Types | Optional | Dynamic |
+|------------|-----------------|-------------------------------------------------------|-----------------|----------|---------|
+| length | The length of a batch. | If length value was not given, it assigns 0 as length and process the whole batch at once. | INT                 | Yes      | No      |
+
+## Example 1
+
+```sql
+CREATE STREAM consumerItemStream (itemId string, price float)
+
+INSERT INTO outputStream
```

Comment:
@StoyanTc and @koshy-thomas, the validator says something is wrong with this query. Please fix it?

Reply:
@diana-macrometa, just add a semicolon to end of CREATE STREAM definition ... price float);... semicolon missing at end of line 21.. this is for validation error.